src/EnergyPlus/DataSurfaces.cc

```diff
@@ -511,11 +511,11 @@ namespace DataSurfaces {
 				edges.push_back( vertices[ ( i + 1 ) % n ] - vertices[ i ] );
 			}
 			if ( shapeCat == ShapeCat::Rectangular ) { // Set side length squared for ray--surface intersection tests
-				assert( n == 4u );
+//				assert( n == 4u );
```

Comment:
As much as I'd like to leave this ... I forgot to uncomment after I debugged a user file. Why are these here anyway? They punish the developer and allow users to make the mistakes these asserts allow in release mode and crash out in debug mode.